modules/olmv1-version-range-support.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-version-range-support_{context}"]
+= Support for version ranges
+
+In {olmv1-first} you can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in an Operator's CR, {olmv1} installs or updates the latest version of the Operator that can be resolved within the version range.
+
+* The resolved version is the latest version of the Operator that satisfies the dependencies and constraints of the Operator and the environment.
+* Operator updates within the specified range are automatically installed if they are resolved successfully.
+* Updates are not installed if they are outside of the specified range or if they cannot be resolved successfully.
```

Comment:
Maybe some sort of list title for these bullets? E.g.:
```suggestion
.Version range considerations
* The resolved version is the latest version of the Operator that satisfies the dependencies and constraints of the Operator and the environment.
* Operator updates within the specified range are automatically installed if they are resolved successfully.
* Updates are not installed if they are outside of the specified range or if they cannot be resolved successfully.
```
or "Version range workflow" or something better.